Fix build error in CI

Looks like we need a try/except around the import, or pick a default `proxy-table` and put it at the expected file path.

```
#17 0.386 > infra-risk-vis@0.2.0 build
#17 0.386 > tsc && vite build
#17 0.386 
#17 11.18 ✘ [ERROR] Could not resolve "./dev-proxy/proxy-table"
#17 11.18 
#17 11.18     vite.config.ts:19:25:
#17 11.18       19 │ import { devProxy } from './dev-proxy/proxy-table'
#17 11.18          ╵                          ~~~~~~~~~~~~~~~~~~~~~~~~~
#17 11.18 
#17 11.18 failed to load config from /app/vite.config.ts
```
from https://github.com/nismod/irv-frontend/actions/runs/6196833311/job/16824248005#step:4:300

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -1,0 +1,24 @@
+name: Run tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+          registry-url: 'https://npm.pkg.github.com'
+          cache: 'npm'
+      - run: npm ci --ignore-scripts
+        env:
+          GH_NPM_AUTH: ${{ secrets.GITHUB_TOKEN }}
+      - run: npm run build
+      - run: npm test

--- a/vite.config.ts
+++ b/vite.config.ts
@@ -16,7 +16,13 @@ import viteTsconfigPaths from 'vite-tsconfig-paths';
  * You can copy and rename one of the included examples.
  * See https://vitejs.dev/config/server-options.html#server-proxy for syntax
  */
-import { devProxy } from './dev-proxy/proxy-table'
+const proxyTable = await import('./dev-proxy/proxy-table').catch((err) => {
+    console.error(err);
+    console.log("Fallback to empty proxy table");
+    return {devProxy: {}}
+});
+const devProxy = proxyTable.devProxy;
+
 
 // https://vitejs.dev/config/
 export default defineConfig({

